pkg/cli/start.go

```diff
@@ -118,9 +119,53 @@ func initInsecure() error {
 	return nil
 }
 
+// maxSizePerProfile is the maximum total size in bytes for profiles per
+// profile type.
+var maxSizePerProfile = envutil.EnvOrDefaultInt64(
```

Comment:
I think I was more concerned with having very little history left.

However, I remembered that we're not talking about goroutine profiles here, which are the ones that can get massive.
A big memory profile is still in the single-digit MB. I don't think cpu is too big either.

So yeah, let's stick with size-based.